lib/forest_liana/collections/customer.rb

```diff
@@ -13,6 +13,11 @@ class Forest::Customer
     isRequired: true,
     description: 'Explain the reason why you want to charge manually the customer here',
     type: 'String'
+  },{
+    field: 'fullname',
+    isRequired: true,
+    description: 'to show how a prefilled fullname will work',
```

Comment:
to show how to pre-fill a field with a smartfield

Reply:
Doesn't looks to be an interesting field for the business/operational point of view.
If we do this anytime we want to show a feature, I think the livedemo will progressively become a real mess.

Reply:
@arnaudbesnier What might be a little more coherent (pertinent business information) is to replace fullname by Stripe_ID. It will look less like a random example just there to show a feature